### PR TITLE
Using fine-tuned model in Kaggle Dog competition

### PR DESCRIPTION
**Pull request by hetong007**
By setting `epoch=30`, one can achieve score ~0.49 on the leaderboard.

@astonzhang Please review.

### CODE REVIEW

File: chapter_computer-vision/kaggle-gluon-dog.md [hunk 1]
```diff
@@ -196,86 +199,49 @@ softmax_cross_entropy = gluon.loss.SoftmaxCrossEntropyLoss()
...
-        out = x
-        for i, b in enumerate(self.net):
-            out = b(out)
-            if self.verbose:
-                print('Block %d output: %s'%(i+1, out.shape))
-        return out
-
+from mxnet.gluon.model_zoo import vision as models
 
 def get_net(ctx):
```
Comment (astonzhang) [inline chapter_computer-vision/kaggle-gluon-dog.md]: Within get_net, we need to add comments to explain, such as line by line (if necessary).

File: chapter_computer-vision/kaggle-gluon-dog.md [hunk 2]
```diff
@@ -149,21 +149,24 @@ from mxnet.gluon.data import vision
...
+                        std=np.array([0.229, 0.224, 0.225]))
     for aug in auglist:
         im = aug(im)
     # 将数据格式从"高*宽*通道"改为"通道*高*宽"。
     im = nd.transpose(im, (2,0,1))
     return (im, nd.array([label]).asscalar().astype('float32'))
 
 def transform_test(data, label):
-    im = image.imresize(data.astype('float32') / 255, 96, 96)
+    im = image.imresize(data.astype('float32') / 255, 256, 256)
```
[the remark this answers is not in the document]
Comment (hetong007) [reply chapter_computer-vision/kaggle-gluon-dog.md]: It's a ImageNet benchmarking standard, we can omit it here for simplicity.

**Comment by hetong007** [comment]
@astonzhang PR is updated accordingly.

**Comment by astonzhang** [comment]
LGTM. Just let me know when you got the performance after 30 epochs.

**Comment by hetong007** [comment]
The full training process uses roughly 4GB GPU ram with batch=128. The accuracy is at 0.446 on the leaderboard.

**Comment by astonzhang** [comment]
Good. Thanks!
